Tenant 'kusto.auth.authority' not found

When running the latest code, the `kusto.auth.authority` property is not properly passed to the connector.

`Caused by: com.microsoft.aad.adal4j.AuthenticationException: {"error_description":"AADSTS90002: Tenant 'kusto.auth.authority' not found. This may happen if there are no active subscriptions for the tenant. Check to make sure you have the correct tenant ID. 
`

--- a/README.md
+++ b/README.md
@@ -39,8 +39,11 @@ docker run --rm -p 3030:3030 -p 9092:9092 -p 8081:8081 -p 8083:8083 -p 8082:8082
 ```
 
 #### Verify 
-connect to container and run:
-`cat /var/log/broker.log /var/log/connect-distributed.log | grep -C 4 i kusto`
+Connect to container and run:
+
+```bash
+cat /var/log/broker.log /var/log/connect-distributed.log | grep -C 4 i kusto
+```
 
 #### Add plugin 
 Go to `http://localhost:3030/kafka-connect-ui/#/cluster/fast-data-dev/` and using the UI add Kusto Sink (NEW button, then pick kusto from list)
@@ -92,25 +95,26 @@ KafkaTest | count
 ```
 
 
-####Supported formats
-csv, json, avro, apacheAvro parquet, orc, tsv, scsv, sohsv, psv, txt.
-> Note - avro, apacheAvro, parquet and orc files are sent each record (file) separately without aggregation, and are expected to be sent as a byte array containing the full file.
-Use value.converter=org.apache.kafka.connect.converters.ByteArrayConverter.
+#### Supported formats
+`csv`, `json`, `avro`, `apacheAvro`, `parquet`, `orc`, `tsv`, `scsv`, `sohsv`, `psv`, `txt`.
+
+> Note - `avro`, `apacheAvro`, `parquet` and `orc` files are sent each record (file) separately without aggregation, and are expected to be sent as a byte array containing the full file.
+> 
+>Use `value.converter=org.apache.kafka.connect.converters.ByteArrayConverter`
 
 
-####Supported compressions
+#### Supported compressions
 Kusto Kafka connector can get compressed data, this can be specified in the topics_mapping in the configuration under 
-'eventDataCompression', this can get all the compression types kusto accepts. Using this configuration files does'nt get 
-aggregated in the connector and are sent straight for ingestion.
+`eventDataCompression`, this can get all the compression types kusto accepts. Using this configuration, files don't get aggregated in the connector and are sent straight for ingestion.
 
 
-####Avro example
+#### Avro example
 One can use this gist [FilesKafkaProducer]("https://gist.github.com/ohadbitt/8475dc9f63df1c0d0bc322e9b00fdd00") to create
 a JAR file that can be used as a file producer which sends files as bytes to kafka. 
-Create an avro file as in `src\test\resources\data.avro`
-copy the jar `docker cp C:\Users\ohbitton\IdeaProjects\kafka-producer-test\target\kafka-producer-all.jar <container id>:/FilesKafkaProducer.jar`
-Connect to the container `docker exec -it <id> bash`.
-Run from the container `java -jar FilesKafkaProducer.jar fileName [topic] [times]`
+* Create an avro file as in `src\test\resources\data.avro`
+* Copy the jar `docker cp C:\Users\ohbitton\IdeaProjects\kafka-producer-test\target\kafka-producer-all.jar <container id>:/FilesKafkaProducer.jar`
+* Connect to the container `docker exec -it <id> bash`.
+* Run from the container `java -jar FilesKafkaProducer.jar fileName [topic] [times]`
 
 ## Need Support?
 - **Have a feature request for SDKs?** Please post it on [User Voice](https://feedback.azure.com/forums/915733-azure-data-explorer) to help us prioritize

--- a/src/main/java/com/microsoft/azure/kusto/kafka/connect/sink/KustoSinkConfig.java
+++ b/src/main/java/com/microsoft/azure/kusto/kafka/connect/sink/KustoSinkConfig.java
@@ -324,27 +324,26 @@ public class KustoSinkConfig extends AbstractConfig {
     }
 
     public String getErrorTolerance() {
-        return this.getString(KUSTO_SINK_TEMP_DIR_CONF);
+        return this.getString(KUSTO_SINK_ERROR_TOLERANCE_CONF);
     }
 
     public String getDlqBootstrapServers() {
-        return this.getString(KUSTO_SINK_TEMP_DIR_CONF);
+        return this.getString(KUSTO_DLQ_BOOTSTRAP_SERVERS_CONF);
     }
 
     public String getDlqTopicName() {
-        return this.getString(KUSTO_SINK_TEMP_DIR_CONF);
+        return this.getString(KUSTO_DLQ_TOPIC_NAME_CONF);
     }
 
     public long getMaxRetryTime() {
-        return this.getLong(KUSTO_SINK_FLUSH_SIZE_BYTES_CONF);
+        return this.getLong(KUSTO_SINK_MAX_RETRY_TIME_MS_CONF);
     }
 
     public long getBackOffTime() {
-        return this.getLong(KUSTO_SINK_FLUSH_SIZE_BYTES_CONF);
+        return this.getLong(KUSTO_SINK_RETRY_BACKOFF_TIME_MS_CONF);
     }
 
     public static void main(String[] args) {
       System.out.println(getConfig().toEnrichedRst());
     }
 }
-

--- a/src/main/java/com/microsoft/azure/kusto/kafka/connect/sink/KustoSinkConfig.java
+++ b/src/main/java/com/microsoft/azure/kusto/kafka/connect/sink/KustoSinkConfig.java
@@ -201,7 +201,7 @@ public class KustoSinkConfig extends AbstractConfig {
             .define(
                 KUSTO_SINK_TEMP_DIR_CONF,
                 Type.STRING,
-                System.getProperty("java.io.tempdir"),
+                System.getProperty("java.io.tmpdir"),
                 Importance.LOW,
                 KUSTO_SINK_TEMP_DIR_DOC,
                 writeGroupName,

--- a/src/main/java/com/microsoft/azure/kusto/kafka/connect/sink/KustoSinkConfig.java
+++ b/src/main/java/com/microsoft/azure/kusto/kafka/connect/sink/KustoSinkConfig.java
@@ -296,7 +296,7 @@ public class KustoSinkConfig extends AbstractConfig {
     }
 
     public String getKustoAuthAppid() {
-        return KUSTO_AUTH_APPID_CONF;
+        return this.getString(KUSTO_AUTH_APPID_CONF);
     }
 
     public String getAuthAppkey() {
@@ -304,7 +304,7 @@ public class KustoSinkConfig extends AbstractConfig {
     }
 
     public String getAuthAuthority() {
-        return KUSTO_AUTH_AUTHORITY_CONF;
+        return this.getString(KUSTO_AUTH_AUTHORITY_CONF);
     }
 
     public String getTopicToTableMapping() {

